owncloudData/src/test/java/com/owncloud/android/data/shares/datasources/OCRemoteShareesDataSourceTest.kt

```diff
@@ -62,54 +59,83 @@ class OCRemoteShareesDataSourceTest {
         } returns getRemoteShareesOperationResult
 
         // Get sharees from remote datasource
-        val sharees = ocRemoteShareesDataSource.getSharees(
+        sharees = ocRemoteShareesDataSource.getSharees(
             "user",
             1,
             30
```

Comment:
I'm not sure at all about this test. `sharees` will be always the same mocked list, what happens when no sharees are retrieved? Is it an empty list or just null? Maybe some edge cases scenarios are missing, what do you think?

Reply:
yes